src/Moq/Language/Flow/SetupSequencePhrase.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.ComponentModel;
 using System.Reflection;
-
+using System.Threading.Tasks;
```

Comment:
Please add a blank line after this so that `using`s are grouped by their main namespace (i.e. one group for `System.*`, one for `Moq.*`).